incuna-sass/helpers/_link.sass

```diff
@@ -1,16 +1,16 @@
 // Mixin to quickly add / remove default styles to links
 @mixin link($type: default)
     cursor: pointer
-    @if $type = default
```

Comment:
Using `@else if` and `@else` is optional, but it matches the code intention better than multiple `@if` statements.